dev/tests/scheduler.rs

```diff
@@ -20,20 +22,24 @@ use plane_dev::{
     timeout::{expect_to_stay_alive, timeout, LivenessGuard},
     util::{base_scheduler_request, random_loopback_ip},
 };
-use std::{net::IpAddr, time::Duration};
-use tokio::time::sleep;
+use std::net::IpAddr;
+use tokio::sync::Mutex;
 
 pub const CLUSTER_DOMAIN: &str = "plane.test";
 const PLANE_VERSION: &str = env!("CARGO_PKG_VERSION");
+lazy_static! {
```

Comment:
I was trying to move more stuff in here, but was having difficulties. open to removing this entirely.